web/src/filter.ts

```diff
@@ -529,6 +529,7 @@ export class Filter {
             "channels-public",
             "channel",
             "topic",
+            "with",
```

Comment:
Is there a reason why `with` was added at this exact index.

Reply:
I though since `with` is associated with `channel` and `topic` operators in the web (every permalink would be a channel, topic and followed by a `with` operator), I placed it right below the `topic` operator.